### PR TITLE
Add EOY mobile snippet

### PR DESCRIPTION
**Pull request by TheoChevalier**

### CODE REVIEW

File: app/config/sources.inc.php [hunk 1]
```diff
@@ -376,9 +377,8 @@
     'snippets/2016/dec2016.lang' => [
         'critical' => ['de', 'es', 'fr', 'pt-BR', 'ru'],
     ],
-    'snippets/2016/dec2016_eoy_a.lang' => [ 'critical' => ['all'] ],
```
Comment (flodolo) [inline app/config/sources.inc.php]: Why did you remove these two? Marking as obsolete would be safer.

Comment (flodolo) [inline app/config/sources.inc.php]: Not really, if you run the script for untracked files you would end up removing them.

Comment (TheoChevalier) [inline app/config/sources.inc.php]: Hum, I’m not sure I follow you, I removed the flag from the files ($lang_flags['engagement']), not the tracking of the files ($engagement_lang). So, untracked_files returns nothing. Am I missing something else?
